Library/Homebrew/formula_installer.rb

```diff
@@ -1129,24 +1130,28 @@ def forbidden_license_check
                                             .to_s
                                             .sub("Public Domain", "public_domain")
                                             .split(" ")
+                                            .to_h do |license|
+      [license, SPDX.license_version_info(license)]
+    end
+
     return if forbidden_licenses.blank?
 
     compute_dependencies.each do |dep, _|
       next if @ignore_deps
 
       dep_f = dep.to_formula
-      next unless dep_f.license.all? { |license| forbidden_licenses.include?(license.to_s) }
+      next unless SPDX.licenses_forbid_installation? dep_f.license, forbidden_licenses
 
       raise CannotInstallFormulaError, <<~EOS
-        The installation of #{formula.name} has a dependency on #{dep.name} where all its licenses are forbidden: #{dep_f.license}.
+        The installation of #{formula.name} has a dependency on #{dep.name} where all its licenses are forbidden: #{SPDX.license_expression_to_string dep_f.license}.
```

Comment:
```suggestion
        The installation of #{formula.name} has a dependency on #{dep.name} where all its licenses are forbidden:
          #{SPDX.license_expression_to_string dep_f.license}.
```

to avoid wrapping. Ideally this won't be over 80 characters in the general case.

Reply:
Is this I rule I should be following? There are a few other of these messages that may not wrap at 80.

Reply:
Yes, ideally.

Reply:
Okay. Should it wrap at 80 or e.g. 73 for the `Error: ` at the beginning?

Reply:
73 in that case, I think. The tricky thing here is unfortunately the `formula.name` and `dep.name` are not consistently the same width so I think it'd be fine if necessary to let this one run on.

The "avoid wrapping" I was mentioning above was "avoid wrapping in the GitHub diff UI"

Reply:
I'm not sure I totally understand, then. Why have a line limit of 118 if you want 80 for text?

Reply:
sorry, I'm saying 
- 118: mandatory to improve readability in the GitHub UI
- 80: optional but ideal to improve readability on the console (but hard to do exactly when lengths are dynamic so just splitting each piece of important information onto a new line is sufficient)

TL;DR: you can use the suggested code and ignore the rest of what I've said here 😉

Reply:
Oh, okay. Thanks for clarifying.